app/controllers/twitter_sign_in.js

```diff
@@ -4,7 +4,7 @@ var oauth = require('../../lib/oauth.js')
 var logger = require('../../lib/logger.js')()
 
 var oauthRequestTokenHandler = function (req, res) {
-  return function (err, oauth_token, oauth_token_secret, results) {
```

Comment:
I have execute standard as it is defined in travis -> npm and it lint only `assets/scripts`. This file is in another location so I'm not getting issue here.

Do we want to change linting check?